lmp/bb-selftest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+source setup-environment build
+
+cat << EOFEOF >> conf/auto.conf
+
+# Disable problematic classes for oe-selftest
```

Comment:
maybe replace problematic by non compatible

Reply:
I think I use problematic because is what bitbake says when they are enabled. Fixed